> I wonder if Zephyr could emit a cmake warning. Something along the lines of:
> 
> ```
> if NATIVE_BUILD and C/CXX_FLAGS contains "-m32" and CMAKE_HOST is (APPLE or LINUX_AARCH64)
>     message(WARNING "Not currently supported because ...")
> endif()
> ```

Our build system already emits such a message:

```
$ west build -b native_sim samples/hello_world/
...
CMake Error at /home/brix/Projects/zephyrproject/zephyr/arch/posix/Linux.aarch64.cmake:28 (message):
  CONFIG_64BIT=n but this Aarch64 machine has a 64-bit userspace.

  If you were targeting native_sim, target native_sim/native/64 instead.

  Otherwise, be sure to define CONFIG_64BIT appropriately.
```

--- a/doc/develop/getting_started/index.rst
+++ b/doc/develop/getting_started/index.rst
@@ -95,6 +95,11 @@ The current minimum required version for the main dependencies are:
               python3-dev python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
               make gcc gcc-multilib g++-multilib libsdl2-dev libmagic1
 
+         .. note::
+
+            Due to the unavailability of ``gcc-multilib`` and ``g++-multilib`` on AArch64
+            (ARM64) systems, you may need to remove them from the list of packages to install.
+
       #. Verify the versions of the main dependencies installed on your system by entering:
 
          .. code-block:: bash

